extra/entrypoint.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env sh
+
+set -e
```

Comment:
Consider adding `set -x`

Reply:
I'm not a big fan of this (in this place). Pls compare `docker logs` without -x :
```
==> Performing startup jobs and maintenance tasks
changed ownership of '/app/data/kuma.db-shm.bak0' from 2005:2005 to 4015:4015
changed ownership of '/app/data/kuma.db-wal.bak0' from 2005:2005 to 4015:4015
changed ownership of '/app/data/kuma.db.bak0' from 2005:2005 to 4015:4015
changed ownership of '/app/data/kuma.db' from 2005:2005 to 4015:4015
changed ownership of '/app/data' from 2005:2005 to 4015:4015
==> Starting application with user 4015 group 4015
Welcome to Uptime Kuma
Node Env: production
Importing Node libraries
Importing 3rd-party libraries
Importing this project modules
Prepare Notification Providers
Version: 1.6.0
Data Dir: ./data/
Creating express and socket.io instance
Server Type: HTTP
Connecting to Database
SQLite config:
[ { journal_mode: 'wal' } ]
[ { cache_size: -12000 } ]
Connected …
```

and with -x parameter set:
```
+ echo ==> Performing startup jobs and maintenance tasks
==> Performing startup jobs and maintenance tasks
+ files_ownership
+ chown -hRc 4025:4025 /app/data
changed ownership of '/app/data/kuma.db-shm.bak0' from 4015:4015 to 4025:4025
changed ownership of '/app/data/kuma.db-wal.bak0' from 4015:4015 to 4025:4025
changed ownership of '/app/data/kuma.db.bak0' from 4015:4015 to 4025:4025
changed ownership of '/app/data/kuma.db' from 4015:4015 to 4025:4025
changed ownership of '/app/data' from 4015:4015 to 4025:4025
==> Starting application with user 4025 group 4025
+ echo ==> Starting application with user 4025 group 4025
+ exec setpriv --reuid 4025 --regid 4025 --clear-groups node server/server.js
Welcome to Uptime Kuma
Node Env: production
Importing Node libraries
Importing 3rd-party libraries
Importing this project modules
Prepare Notification Providers
Version: 1.6.0
Data Dir: ./data/
Creating express and socket.io instance
Server Type: HTTP
Connecting to Database
SQLite config:
[ { journal_mode: 'wal' } ]
[ { cache_size: -12000 } ]
Connected …
```
It adds too much "garbage info" to the log and make it unclear for user to inspect. Now it's not a big deal, but if there are any other startup functions in a future it could be look only worse.
Paraphrasing Otto von Bismarck _"Logs are like sausages, it is better not to see them being made."_ 

The optimal solution would be to introduce `DEBUG` environment variable. If set to `true` then set `-x`. It could also set more verbose messages from Node. But IMO it's an idea for quite separate PR.